virttest/qemu_vm.py

```diff
@@ -1864,19 +1864,28 @@ def sort_key(dev):
 
         # Add pvpanic device
         if params.get("enable_pvpanic") == "yes":
-            if not devices.has_device("pvpanic"):
+            if 'aarch64' in params.get('vm_arch_name', arch.ARCH):
+                pvpanic = 'pvpanic-pci'
+            else:
+                pvpanic = 'pvpanic'
+            if not devices.has_device(pvpanic):
                 logging.warn("pvpanic device is not supportted")
```

Comment:
```suggestion
                logging.warn("%s device is not supported", pvpanic)
```
It could prompt more clearly which device is not supported.
Like:
aarch64:
`"pvpanic-pci device is not supported"`
others:
`"pvpanic device is not supported"`

BTW, there is a typo of `supportted`  -> supported

Reply:
Thanks a lot, Corrected.
Testing again